source/process/sg_document-structure.rst

```diff
@@ -0,0 +1,30 @@
+==================
+Document structure
+==================
+
+Structure and organization are an important part of a document's ease of use and its understandability. Information should be organized and presented in a logical order, with similar subjects grouped together in the same section.
+
+In most cases, a document has a title, an introductory paragraph, and one or more sections.
+
+Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate.
+
+Document title
+==============
+
+Use a title that accurately reflects the content of the document. People scan the table of contents looking for answers; it's often faster than using the built-in search engine.
+
+Use title case for document titles. For more information and an example of capitalization, see :ref:`capital`.
+
+Introductory paragraph
+======================
+
+Each page should have an introduction that acts as a short description of the document. The short description should be a single paragraph of no more than 3 sentences. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want.
+
+Document sections
+=================
+
+To make pages easier for people to quickly scan for the content that they're looking for, break your document up into logical sections. Each section should have a title, and the title should relate to the content of the section.
+
+A section title is not required if you have only one section.
+
+Avoid subsections. If you find that you need subsections, quite often the document is too long, or too complex, and needs to be broken up into several pages.
```

Comment:
I kept the pages separate, but when built they're all included into a monolith, which is what readers will see. See sg_mattermost-doc-style.rst for the `..include::` statements.

Here's an idea of what it would look like when ouput to HTML:

![image](https://cloud.githubusercontent.com/assets/20325173/20247879/e2de9d6e-a98c-11e6-8a6f-f87f91c89e39.png)

The *Document sections* stuff is in `sg_document-structure.rst` and *Grammar, spelling, mechanics* is in `sg_grammar-spelling-mechanics.rst`.